proxy/http_headers_test.go

```diff
@@ -235,6 +235,19 @@ func TestAddHeaders(t *testing.T) {
 			},
 			"",
 		},
+
+		{"set uniquerequestid header",
+			&http.Request{RemoteAddr: "1.2.3.4:5555", Host: "5.6.7.8:1234"},
+			config.Proxy{UniqueRequestID: "X-Unique-Request-Id"},
+			http.Header{
+				"Forwarded":           []string{"for=1.2.3.4; proto=http"},
+				"X-Forwarded-Proto":   []string{"http"},
+				"X-Forwarded-Port":    []string{"1234"},
+				"X-Real-Ip":           []string{"1.2.3.4"},
+				"X-Unique-Request-Id": []string{""}, // XXX nedd to figure out how to regexp match
```

Comment:
Logging can be made by using $header.ConfiguredHederName format already present in fabio.